### DISCUSSION TITLE
Example of logging support

### DISCUSSION THREAD

**Discussion by davhdavh** [microsoft/dotnet-framework-docker#1106] [Q&A]
### Describe the Problem

It is unclear how to configure logging in the .net framework application, so that it gets output as normal docker-style logging.

None of the samples use logging.

### Describe the Solution

Just a single log message from inside the container ending up as docker-style log is enough.

**Comment by lbussell** [comment]
Hi @davhdavh, I'm not sure what you're asking for. `docker logs <your container name>` should be able to get you the console output from your application. I have had no issues getting output directly with `Console.WriteLine(...)`. Are you looking for something more sophisticated with `Microsoft.Extensions.Logging`?

**Reply by davhdavh** [reply]
For dotnet core, sure, trivial.
For asp.net framework apps running under iis in docker there doesn't seem to be a simple solution.

**Reply by lbussell** [reply]
@John-Hart, can you provide any input on logging for IIS?

**Reply by John-Hart** [reply]
You might look at https://techcommunity.microsoft.com/t5/containers/announcing-log-monitor-2-0-stable-release-and-2-1-release/ba-p/3883870
